src/setupTest.js

```diff
@@ -24,3 +24,15 @@ process.env.LOGO_URL = 'https://edx-cdn.org/v3/default/logo.svg';
 process.env.LOGO_TRADEMARK_URL = 'https://edx-cdn.org/v3/default/logo-trademark.svg';
 process.env.LOGO_WHITE_URL = 'https://edx-cdn.org/v3/default/logo-white.svg';
 process.env.FAVICON_URL = 'https://edx-cdn.org/v3/default/favicon.ico';
+process.env.ABOUT_US_URL = 'http://localhost:18000/about';
+process.env.PRIVACY_POLICY_URL = 'http://localhost:18000/privacy';
+process.env.HONOR_CODE_URL = 'http://localhost:18000/tos_and_honor';
+process.env.TERMS_OF_SERVICE_URL = 'http://localhost:18000/tos_and_honor';
+process.env.CONTACT_URL = 'http://localhost:18000/about';
+process.env.SUPPORT_CENTER_URL = 'http://localhost:18000/about';
```

Comment:
Contact and Support_Central URL mean to point same ?

Reply:
The Contact Us and Support Center have been given dummy values so that they could render in the snapshot (as links dint render if not provided). So that the code coverage could pass the acceptance criteria.
These re dummy values just for testing.